rust/src/http2/http2.rs

```diff
@@ -647,7 +647,7 @@ impl HTTP2State {
                         //we could set an event on remaining data
                         return HTTP2FrameTypeData::SETTINGS(set);
                     }
-                    Err(nom::Err::Incomplete(_)) => {
+                    Err(Err::Incomplete(_)) => {
```

Comment:
Why not keep `nom7::Err::Incomplete(_)` ?

Reply:
To simplify later updates by regrouping all namespaces at top of file, so if we change back the name from `nom7` to `nom` this will be much easier.
Note: the `nom7` name is only a trick used to simplify the transition

Reply:
I was rather afraid of collisions about `Err` from something else... What do you think ?

Reply:
In my experience, rust handles this quite well: it resolves the type `Err::Incomplete` as a variant from the nom type, without conflict with the builtin variant `Result::Err`, so the notation will not create any conflict.
The only possible error would be to explicitly import several types `Err` in the `use` statements, but the compiler detects that and will require to change one of the names.